src/library/scala/collection/generic/IsIterableLike.scala

```diff
@@ -87,55 +83,70 @@ import scala.reflect.ClassTag
  * instance of `IsIterableLike` specific to the new type.
  *
  * Below is an example of an implementation of the `IsIterableLike` trait
- * where the `Repr` type is `String`.
+ * where the `Repr` type is `Range`.
  *
  *{{{
- * implicit val stringRepr: IsIterableLike[String] { type A = Char } =
- *   new IsIterableLike[String] {
- *     type A = Char
- *     val conversion = implicitly[String => IterableOps[Char, Any, String]]
+ * implicit val rangeRepr: IsIterableLike[Range] { type A = Int; type C = IndexedSeq[Int] } =
+ *   new IsIterableLike[Range] {
+ *     type A = Int
+ *     type C = IndexedSeq[Int]
+ *     def apply(coll: Range): IterableOps[Int, Iterable, IndexedSeq[Int]] = coll
  *   }
  *}}}
  */
 trait IsIterableLike[Repr] {
-  /** The type of elements we can traverse over. */
+
+  /** The type of elements (e.g. `Int`) */
   type A
-  /** A conversion from the representation type `Repr` to `IterableOps[A, Iterable, Repr]`. */
-  val conversion: Repr => IterableOps[A, Iterable, Repr]
```

Comment:
`IsTraversableLike` in 2.12 calls it `conversion`.

Reply:
Yeah, I can call it back `conversion`. I think I initially made it a method following up [that comment](https://github.com/scala/collection-strawman/pull/478#discussion_r171077953).